x-pack/test/functional/apps/monitoring/enable_monitoring/index.js

```diff
@@ -46,8 +47,10 @@ export default function ({ getService, getPageObjects }) {
       });
 
       // Here we are checking that once Monitoring is enabled,
-      //it moves on to the cluster overview page.
-      await retry.try(async () => {
+      // it moves on to the cluster overview page.
+      await retry.tryForTime(10000, async () => {
```

Comment:
nit: Ten seconds seems kind of long. Is there any way to add [Promise.race()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/race) condition? Maybe OR if a certain element is visible?

But, I think it's still fine if not